package.json

```diff
@@ -27,7 +27,8 @@
   "dependencies": {
     "brcast": "^1.1.6",
     "html-tag-names": "^1.1.1",
-    "svg-tag-names": "^1.1.0"
+    "svg-tag-names": "^1.1.0",
+    "react-html-attributes": "^1.0.2"
```

Comment:
Any chance you could update the `yarn.lock` file as well? You'll need [`yarn`](https://yarnpkg.com) installed, and then you just run `yarn` and it should update itself.